app.go

```diff
@@ -555,6 +555,11 @@ func New(config ...Config) *App {
 	return app
 }
 
+func (app *App) updateStack() {
```

Comment:
we can rename the method and make it more useful by adding the extension of the intMethod list here as well

perhaps rename it to 
addCustomRequestMethod
and the parameter is the name of the new request method, after that the other 2 lists will be expanded